src/tests/ftest/launch.py

```diff
@@ -2336,6 +2342,54 @@ def _reset_server_storage(test):
             logger.debug("  Skipping resetting server storage - no server hosts")
         return 0
 
+    @staticmethod
+    def _cleanup_procs(test, do_stop):
```

Comment:
@ashleypittman This function is primarily for dfuse processes/mounts that might hang around if a test times out and doesn't finish cleanup. The commands ran are essentially:
`sudo -n pkill -e --signall KILL dfuse`
`sudo -n umount -v --all --force -t daos.fuse`
This is meant to be heavy-handed. Is there anything else we should do?

Reply:
It really depends on how dfuse is stuck, if there's I/O in flight for example and dfuse isn't responding as it can't communicate with the server then it's hard to kill.  You can also get the case where the mount is active but dfuse has exited but it looks like this PR covers that.

I think this is reasonable but we should keep an eye on it and improve if we find any escapes.

Reply:
Also, for what it's worth you could also put this function on L2096 (indented one more level) and have it as a private function rather than a static class method.

Reply:
> I think this is reasonable but we should keep an eye on it and improve if we find any escapes.

Definitely! Thanks.

> put this function on L2096 (indented one more level) and have it as a private function rather than a static class method

This function is called from two different functions, so it has to be at the class level